spacebench/algorithms/dapsm.py

```diff
@@ -0,0 +1,115 @@
+import pandas as pd
+import numpy as np
+import sys
+from scipy.optimize import linear_sum_assignment
```

Comment:
We follow [PEP8](https://peps.python.org/pep-0008/) styling guide. So the imports should be categorized into three sections:

- Standard library imports
- Related third party imports
- Local application/library specific imports

There should be a space between each group and you can sort them based on their length.

```
import sys

import numpy as np
import pandas as pd
from scipy.optimize import linear_sum_assignment